src/journal/journalctl.c

```diff
@@ -1460,13 +1460,19 @@ static int list_boots(sd_journal *j) {
         if (count == 0)
                 return count;
 
-        table = table_new(OUTPUT_MODE_IS_JSON(arg_output) ? "index" : "idx", "boot id", "first entry", "last entry");
+        table = table_new("idx", "boot id", "first entry", "last entry");
         if (!table)
                 return log_oom();
 
         if (arg_full)
                 table_set_width(table, 0);
 
+        if (OUTPUT_MODE_IS_JSON(arg_output)) {
```

Comment:
drop the conditionalization. You can call table_set_json_field_name() in any case. If the json output mode is not selected it of course won't actually do anything, but it doesn't hurt either.

Reply:
Ah, that's neat, thanks!